tests/build.gradle

```diff
@@ -60,6 +60,13 @@ test {
         systemProperty 'updateTest262properties', System.getProperty('updateTest262properties')
     }
 
+    System.properties.each { k,v->
+        if (k.startsWith("rhino.")) {
+            systemProperty k, v
+        }
+    }
```

Comment:
I copy all rhino system properties to the sub process. So you can pass properties with `-Drhino.xyz` from a gradle buld